tests/phpunit/src/Application/ExceptionApplicationTest.php

```diff
@@ -14,21 +16,27 @@
  *
  * @package Acquia\Cli\Tests\Application
  */
-class ExceptionApplicationTest extends ApplicationTestBase {
+class ExceptionApplicationTest extends TestBase {
 
-  public function setUp($output = NULL): void {
+  public function setUp($output = NULL):void {
+    // If kernel is cached from a previous run, it doesn't get detected in code
+    // coverage reports.
+    shell_exec('rm -rf var/cache');
```

Comment:
Can we use the Symfony filesystem class for this rather than running `shell_exec()` ?

Reply:
Thanks, that's definitely less of a hack. Done.